files/zh-cn/web/http/caching/index.html

```diff
@@ -148,14 +148,14 @@ <h2 id="Vary_响应">Vary 响应</h2>
 
 <pre class="line-numbers  language-html notranslate"><code class="language-html">Vary: User-Agent</code></pre>
 
-<p>因为移动版和桌面的客户端的请求头中的 User-Agent 不同， 缓存服务器不会错误地把移动端的内容输出到桌面端到用户。</p>
+<p>因为移动版和桌面的客户端的请求头中的 User-Agent 不同，缓存服务器不会错误地把移动端的内容输出到桌面端到用户。</p>
 
 <h2 id="参考">参考</h2>
 
 <ul>
  <li><a href="https://tools.ietf.org/html/rfc7234">RFC 7234: Hypertext Transfer Protocol (HTTP/1.1): Caching</a></li>
  <li><a href="https://www.mnot.net/cache_docs">Caching Tutorial – Mark Nottingham</a></li>
- <li><a href="https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/http-caching">HTTP caching – Ilya Grigorik</a></li>
+ <li><a href="https://developers.google.com/Web/fundamentals/performance/optimizing-content-efficiency/http-caching">HTTP caching – Ilya Grigorik</a></li>
```

Comment:
Leads to `404`, just remove this link. If you can, please update the `See Also` section (see: https://developer.mozilla.org/en-US/docs/Web/HTTP/Caching)

![image](https://user-images.githubusercontent.com/15844309/171999661-a245bf33-8eaf-4a86-925d-3bdebb6cac5e.png)